core/src/main/java/com/netease/arctic/op/OverwriteBaseFiles.java

```diff
@@ -118,14 +128,17 @@ protected StructLikeMap<Long> apply(Transaction transaction, StructLikeMap<Long>
       partitionData = keyedTable.spec().isUnpartitioned() ? null : d.partition();
       partitionMaxTxId.put(partitionData, getPartitionMaxTxId(partitionData));
     }
-    // 为什么？
+
     for (DataFile d : this.deleteFiles) {
       overwriteFiles.deleteFile(d);
       partitionData = keyedTable.spec().isUnpartitioned() ? null : d.partition();
       partitionMaxTxId.put(partitionData, getPartitionMaxTxId(partitionData));
     }
     if (transactionId != null && transactionId > 0) {
-      overwriteFiles.set("txId", transactionId + "");
+      overwriteFiles.set(PROPERTIES_TRANSACTION_ID, transactionId + "");
+    }
+    if (properties != null) {
```

Comment:
The logic is repetitive with line 144-146

Reply:
yes, i would drop it